### PR TITLE
Notebooks to explore how the site api works

### PR DESCRIPTION
**Pull request by zakwatts**
A notebook for clients and an additional notebook for testing production and development forecasts have been created.

These notebooks allow you to:
- Get your access token for clients hitting an endpoint with user credentials and from a .env file for prod/dev testing.
- See data from sites that are set to have forecasts generated
- Pull forecasts from specific sites
- Plot forecasts
- Pull multiple forecasts from a list of sites and plot.

### CODE REVIEW

File: notebooks/dev_prod_sites_api_explore.ipynb [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+    "    if environment == \"prod\":\n",
+    "        base_url = \"https://api-site.quartz.solar\"\n",
+    "        client_id = \"QLMdXCCHMS9Zl5W2tDG3otpdcY7GeEnJ\"\n",
+    "        domain = \"nowcasting-pro.eu.auth0.com\"\n",
+    "        access_token = os.getenv('PROD_ACCESS_TOKEN')\n",
+    "        if access_token is None:\n",
+    "            raise ValueError(\"PROD_ACCESS_TOKEN not found in environment variables\")\n",
+    "    elif environment == \"dev\":\n",
+    "        base_url = \"http://uk-development-sites-api.eu-west-1.elasticbeanstalk.com\"\n",
+    "        client_id = \"QLMdXCCHMS9Zl5W2tDG3otpdcY7GeEnJ\"\n",
```
Comment (peterdudfield) [inline notebooks/dev_prod_sites_api_explore.ipynb:68]: should these client ids be different, should they maybe even be in .env file?

Comment (peterdudfield) [reply notebooks/dev_prod_sites_api_explore.ipynb:68]: actually i think its fine they are there

**Review by peterdudfield — CHANGES_REQUESTED** [review]
one small change, but looks great

File: notebooks/dev_prod_sites_api_explore.ipynb (hunk 1 above)
Comment (zakwatts) [reply notebooks/dev_prod_sites_api_explore.ipynb:68]: Yes, so its the same client_id but a different base_url. I'll just set it once.

**Review by peterdudfield — APPROVED** [review]

**Pull request merged**